e2etest/pkg/config/update.go

```diff
@@ -152,7 +163,26 @@ func (o beta1Updater) poolToCR(a AddressPool) *operatorv1beta1.AddressPool {
 }
 
 func (o beta1Updater) createPeer(p Peer) error {
-	peer, err := o.peerToCR(p)
+	secretName := fmt.Sprintf("bgppeer-%d-%d-secret", p.ASN, p.Port)
+	existing := &corev1.Secret{}
+	err := o.Client.Get(context.TODO(), types.NamespacedName{Name: secretName, Namespace: o.namespace}, existing)
```

Comment:
We need to ensure both work, so I'd add a specific test that just fills the secret and checks the session is established (as opposed to changing all the tests)

Reply:
yes, added specific e2e tests to cover this.